docs/README.md

```diff
@@ -74,28 +75,34 @@ The extra `update` permission is to update the `GSLBConfig` and `GlobalDeploymen
    $ helm search repo
 
    NAME     	CHART VERSION    	APP VERSION      	DESCRIPTION
-   ako/amko	1.4.1	            1.4.1	            A helm chart for Avi Multicluster Kubernetes Operator
+   ako/amko	1.4.2	            1.4.2	            A helm chart for Avi Multicluster Kubernetes Operator
 
    ```
 
 4. Use the `values.yaml` from this repository to provide values related to Avi configuration. To get the values.yaml for a release, run the following command
 
    ```
-   helm show values ako/amko --version 1.4.1 > values.yaml
+   helm show values ako/amko --version 1.4.2 > values.yaml
 
    ```
    Values and their corresponding index can be found [here](#parameters)
 
-5. Install AMKO:
+5. On the leader cluster, set the following fields in `values.yaml`:
+   * Set `configs.federation.currentClusterIsLeader` to `true` only on the leader cluster. For all other clusters, set this to `false`.
```

Comment:
you have already say in line 90 "set the following fields"
you don't have to again repeat "Set `configs`..." on the bullets.
OR

You could just say - "On the leader cluster" in Line 90 and keep the content as is.